### PR TITLE
GetSubmission and GetSubmissions - Adding Prefer header and expand options in Examples.

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**
the purpose of the PR is;

- to mention the `prefer` request header in both pages.
- show example of how to use $expand options in GetSubmission.

### CODE REVIEW

File: api-reference/beta/api/educationassignment-list-submissions.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ List all the submissions associated with an [assignment](../resources/educationa
 
 A teacher or an application with application permissions can get all the submissions while a student can only get submissions that they are associated with.
 
+If the header `Prefer: include-unknown-enum-members` is provided, reassigned submissions retain the `reassigned` status. For details, see the examples section.
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationassignment-list-submissions.md:20]: I'd suggest moving this line to the headers section, below the table.

File: api-reference/beta/api/educationassignment-list-submissions.md [hunk 2]
```diff
@@ -296,6 +299,135 @@ Content-length: 4492
 }
 ```
 
+### Example 3: Get submissions - Request with optional Prefer header
+#### Request
+The following is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationassignment-list-submissions.md:306]: Line 306: please remove this line. Snippets are automatically generated.

Comment (isvargasmsft) [reply api-reference/beta/api/educationassignment-list-submissions.md]: I fixed this one already :)

File: api-reference/beta/api/educationassignment-list-submissions.md (hunk 1 above)
Comment (isvargasmsft) [inline api-reference/beta/api/educationassignment-list-submissions.md:20]: Please clarify why this warning is important for a list operation.

File: api-reference/beta/api/educationsubmission-get.md [hunk 3]
```diff
@@ -21,6 +21,8 @@ Only the **assignedTo** student can see and modify the submission. A teacher or
 
 The grade and feedback from a teacher are part of the [educationOutcome](../resources/educationoutcome.md) associated with this object. Only teachers or applications with application permissions can add or change grades and feedback. Students will not see the grade or feedback until the assignment has been released.
 
+If the header `Prefer: include-unknown-enum-members` is provided, a reassigned submission retains the `reassigned` status. For details, see the examples section.
```
Comment (isvargasmsft) [inline api-reference/beta/api/educationsubmission-get.md:24]: What do you mean by "retains"? Please clarify the importance of this warning in a Get operation.

File: api-reference/beta/api/educationassignment-list-submissions.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/beta/api/educationassignment-list-submissions.md:20]: Clarified! Thank you!

File: api-reference/beta/api/educationsubmission-get.md (hunk 3 above)
Comment (isvargasmsft) [reply api-reference/beta/api/educationsubmission-get.md:24]: Clarified, thank you!

File: api-reference/beta/resources/educationsubmission.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ If [setUpResourcesFolder](../api/educationsubmission-setupResourcesFolder.md) ha
 |returnedBy|[identitySet](identityset.md)|User who moved the status of this submission to returned.|
 |returnedDateTime|DateTimeOffset|Moment in time when the submission was returned. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|
 |resourcesFolderUrl|String|Folder where all file resources for this submission need to be stored.|
-|status|string| Read-only. Possible values are: `working`, `submitted`, `released`, `returned`, and `reassigned`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `reassigned`.|
+|status|string| Read-only. Possible values are: `working`, `submitted`, `released`, `returned`, `unknownFutureValue` and `reassigned`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `reassigned`.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/educationsubmission.md:45]: I noticed this description was missing `unknownFutureValue`. Please check if this is an isolated error or if other properties must be updated. 

In addition, the type says string, but this enum is called educationSubmissionStatus. Please use the enum name in the type column. And also check if this is an isolated issue or if other properties of this API must be updated.

File: api-reference/beta/api/educationassignment-list-submissions.md [hunk 5]
```diff
@@ -17,6 +17,8 @@ List all the submissions associated with an [assignment](../resources/educationa
```
[the remark this answers is not in the document]
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationassignment-list-submissions.md:17]: done

File: api-reference/beta/resources/educationsubmission.md (hunk 4 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/resources/educationsubmission.md:45]: isolated case.
property updated

**Pull request closed without merging**